src/nodes.jl

```diff
@@ -0,0 +1,95 @@
+using SparseArrays
```

Comment:
Do we still need this?

Reply:
Yes, this is still required, see line 73. We still handle sparse matrices for non-precompiled kernels.

Reply:
oh, I mean that we already had the `using` statements collected in the file `LIBSVM.jl`

Reply:
I did not notice. I removed the one in `LIBSVM.jl` since it is not used in that file, and kept the one in `nodes.jl`.